ncaa_vb_pbp.R

```diff
@@ -119,12 +118,19 @@ game_id = 5168298
     ) %>%
     fill(score, .direction = "down") %>%
     separate(score, into = c("away_score", "home_score"), extra = "merge") %>%
-    groupby()
-
-  View(plays)
-
-
-
-  # return(data.frame())
-# }
+    group_by(rally_number) %>%
+    mutate(
+      rally_play_number = row_number()
+    ) %>%
+    ungroup() %>%
+    mutate(
+      rally_play_number = case_when(
+        grepl("Sub", action_type) == TRUE ~ as.integer(NA),
```

Comment:
incredibly minor thing but instead of `as.integer(NA)` you can just do `NA_integer_`

Reply:
also incredibly minor but no need to check `== TRUE`, ` grepl("Sub", action_type) == TRUE` is identical to `grepl("Sub", action_type)`